src/js/components/AppListLabelsFilterComponent.jsx

```diff
@@ -72,7 +72,11 @@ var AppListLabelsFilterComponent = React.createClass({
           }
         });
         return memo;
-      }, []);
+      }, [])
+      .sort((a, b) => {
+        let [labelKeyA, labelKeyB] = [Object.keys(a)[0], Object.keys(b)[0]];
```

Comment:
Suggestion for a refactoring in another PR.
It would be much easier to work with an array-pair instead of a key-value-object.
Also under the aspect that this key-value-object is always converted to an array, if it's used. :)

The representation of the labels in this component here would look like this:
```javascript
[
  ["key1", "value1"],
  ["key2", "value2"]
]
```
instead of this like we have now:
```javascript
[
  {"key1": "value1"},
  {"key2": "value2"}
]
```

That would make the things much easier.
Also this sorting function here.